hopsworks-web/yo/app/views/newStorageConnector.html

```diff
@@ -321,13 +339,94 @@
                         </div>
                     </div>
 
+                    <div class="view-info"
+                         ng-if="newStorageConnectorCtrl.storageConnectorType === 1 && !newStorageConnectorCtrl.s3IAMRole">
```

Comment:
These options should be available regardless the fact that the user has specified an IAM Role